modules/gateways/blockonomics/usdtblockonomics.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require_once __DIR__ . '/../../../init.php';
+require_once __DIR__ . '/../../../includes/gatewayfunctions.php';
+require_once __DIR__ . '/../../../includes/invoicefunctions.php';
+require_once __DIR__ . '/blockonomics.php';
+
+use Blockonomics\Blockonomics;
+
+
+// Init Blockonomics class
+$blockonomics = new Blockonomics();
+$systemUrl = \App::getSystemURL();
+
+function process_finish_order($finish_order, $crypto, $txid)
+{
+    if ($crypto !== "usdt") {
+        return;
```

Comment:
Since we now check if the crypto is _usdt_ before calling _process_finish_order_, we can remove this check.